### DISCUSSION TITLE
"Selected Mods" filter on local leaderboards seems to have no effect

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#21320] [Q&A] [answered]
Selected Mods toggled off
![image](https://user-images.githubusercontent.com/102317848/202875249-a3d437e5-2978-457a-987d-e109140f2144.png)

Selected Mods toggled on
![image](https://user-images.githubusercontent.com/102317848/202875261-8b056e2a-5804-4624-9fa5-4bfd90866fd8.png)

update: it seems to display scores which include your currently selected mods, but not necessarily containing **only** your selected mods.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/21322
